src/tests/test_config.py

```diff
@@ -22,6 +22,8 @@
 from qat.purr.compiler.devices import QuantumComponent
 from .qasm_utils import get_test_file_path, TestFileType, get_qasm2
 
+SUPPORTED_CONFIG_VERSIONS = ["v02", "v1"]
```

Comment:
It's not a different version of the config, its legacy namespacing. Just name it 'legacy' instead.